templates/overrides/navigation/block--system-menu-block--main.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Theme override of the system menu b
```

Comment:
```suggestion
 * Theme override of the system menu block.
 *
 * Removes the outer <nav> wrapper, because the block is placed in a region that is already inside a <nav> tag.
```